plans/SFC.md

```diff
@@ -0,0 +1,25 @@
+Mikkel Kim - mck782
+Hongbing Cai - hc8453
+Jungmo Ku - jk36326
+
+EVERYONE HAS TO LEARN SCALA FIRST.
```

Comment:
please let me know if this becomes an obstacle.  by this point, you should be able to read scala code and be able to make minor changes.